src/main/java/my/cake/shop/Inventory.java

```diff
@@ -0,0 +1,56 @@
+package my.cake.shop;
+import lombok.Getter;
+
+@Getter
+public class Inventory {
+
+  private static final int CARTON_OF_EGGS = 12;
+  private static final int BAG_OF_FLOUR = 2000;
+  private static final int BAG_OF_SUGAR = 1000;
+  private static final int SUGAR_PRICE = 13;
+  private static final int EGGS_PRICE = 10;
+  private static final int FLOUR_PRICE = 30;
+  private int eggs;
+  private int flour;
+  private int sugar;
+  private int restockCost;
+
+  public Inventory() {
+    this.eggs = 12;
+    this.flour = 2000;
+    this.sugar = 1000;
+  }
+
+  public int getIngredientsForCake(String cakeSize) {
+    if (cakeSize.equals("small")) {
```

Comment:
The cakeSize is a string and there's no validation, what would happen if customer had sizePreference set to "Small" instead of "small"?